test/activities/quizzes/QuizEntity.js

```diff
@@ -728,4 +728,18 @@ describe('QuizEntity', () => {
 			});
 		});
 	});
+
+	describe('Has Attempts Completed', () => {
```

Comment:
nit. Can you put this under the `submissionViewsHref` test instead since that's the order of the code you're testing as well?